.eslintrc.json

```diff
@@ -2,7 +2,8 @@
   "env": {
     "node": true,
     "es2021": true,
-    "jest": true
+    "jest": true,
+    "browser": true
```

Comment:
Por que vcs colocaram `"browser": true` aqui? A gente ta no backend então teoricamente não é pra ter nenhuma variável global do browser.

Mas isso é teoricamente né kajdjkasd vai q, esse mundo é doido mesmo

Reply:
é pq ele não tava deixando colocar o tipo "file" na interface, tava dando que "file is not defined" daí foi essa forma que encontramos de resolver, mas não sei se teria outro jeito tbm, não sei

Reply:
Aguardando atualização do arquivo .eslintrc.json

Reply:
Acho que ta certo mesmo @lorenaaze. Parece que é um problema mais com o eslint do que com typescript. N sei tbm. Mas acho que por enquanto pode deixar o `browser true` ali mesmo

Reply:
A opção que o VScode dava era colocar "// eslint-disable-next-line no-undef" antes de colocar o "upload: file", daí não sei como seria melhor deixar